gemet/thesaurus/management/commands/importspreadsheet.py

```diff
@@ -95,6 +96,15 @@ def _create_concepts(self, sheet):
                         name=name,
                         value=value,
                     )
+                published_property = models.Property.objects.filter(
+                    concept=concept,
+                    language=self.language,
+                    name=name,
+                    status=PUBLISHED
+                ).first()
+                if published_property:
+                    published_property.status = DELETED_PENDING
+                    published_property.save()
```

Comment:
Fetch the property once (`status__in=[PUBLISHED, PENDING]`) - always try to keep the number of queries to a minimum. If the property doesn't exist, create it; else if status is `PENDING` modify its value, else if status is `PUBLISHED`, modify its status and create the new pending property.